simcon_project/templates/create_assignment.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
```

Comment:
I think this should either extend the researcher nav bar or link back to the assignment management page after I've merged it into main.